cms/djangoapps/contentstore/management/commands/reindex_course.py

```diff
@@ -129,6 +135,20 @@ def handle(self, *args, **options):  # pylint: disable=too-many-statements
             course_keys = list(map(lambda course: course.id, active_courses))
 
             logging.warning(f'Selected {len(course_keys)} active courses over a total of {len(all_courses)}.')
+        elif inclusion_date_option:
+            # in case of --from_inclusion_date, we get the list of course keys from all courses
+            # that are stored in modulestore and filter out courses with a start date less than
+            # the settings defined COURSEWARE_SEARCH_INCLUSION_DATE
+            all_courses = modulestore().get_courses()
```

Comment:
I am concerned about the performance of this call. Would this take a long time? Is there a way we can try to test out the return time of this call before the PR is merged?

Reply:
Talked about this offline, but this same code is used in two other places in this command: 
- https://github.com/openedx/edx-platform/blob/4f680258e0bf04715f8cf2dafafe05f5ca7a7557/cms/djangoapps/contentstore/management/commands/reindex_course.py#L121
- https://github.com/openedx/edx-platform/blob/4f680258e0bf04715f8cf2dafafe05f5ca7a7557/cms/djangoapps/contentstore/management/commands/reindex_course.py#L127